pkg/csi/service/vanilla/controller.go

```diff
@@ -514,53 +456,31 @@ func (c *controller) ReloadConfiguration() error {
 		}
 		if newVCConfig != nil {
 			newVCConfig.ReloadVCConfigForNewClient = true
+			if oldvCenter != newVCConfig.Host {
+				log.Infof("Observed new vCenter server: %q in the config secret. "+
```

Comment:
`Observed new vCenter server` can be changed to `Observed a change in vCenter server details`

Reply:
No this is for a new vCenter server. Not a change in the existing vCenter server.

Reply:
@divyen, you mean a new vCenter server has been added to the deployment and you are not referring to the case where IP changes to FQDN?

Reply:
@skogta we are restarting containers when VC Host changes, regardless of multi VC feature is enabled or disabled.

Reply:
I thought this code block is for Single VC. For single VC there is no new vCenter right ?

Reply:
we can change VC IP to FQDN, which is considered as new VC.